commands/group.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
                 existing = await bot.db.getGroup(msg.author.id, name);
                 if(existing) return "You already have a group with that name.";
                 await bot.db.addGroup(msg.author.id, bot.noVariation(name));
-                return `Group created. Add ${cfg.lang}s to it with '${cfg.prefix}group add ${name} <name>'.`;
+                return `Group created. Add ${cfg.lang}s to it with '${cfg.prefix}group add ${args.slice(1).length > 1 ? name : "'" + name + "'"} <name>'.`;
```

Comment:
```suggestion
                return `Group created. Add ${cfg.lang}s to it with '${cfg.prefix}group add ${args.length < 2 ? name : "'" + name + "'"} <name>'.`;
```
Avoids making a new array for the slice, also the check was backwards